### DISCUSSION TITLE
Is this repo still maintained?

### DISCUSSION THREAD

**Discussion by dfoxg** [dswd/vpncloud#351] [General]
It looks like this repo is no longer maintained, since the last commit was in 2021. Are there any plans to continue the support? I love vpncloud; simple and functional🎉

**Comment by dfoxg** [comment]
Hi @dswd did you saw that thread?

**Reply by dswd** [reply]
Hi there, sorry I did not see that comment.
I did not abandon VpnCloud but I don't have much time for development either. Thankfully VpnCloud just works as is.
I will from time to time update the dependencies and fix bugs as they appear but I currently don't find the time to work on the plans I have. I was in the middle of migrating to a multi-threaded core that would allow to integrate a control socket into the daemon and allow to monitor/configure it at runtime and also build a frontend for it, etc. but that is WIP since two years.

**Reply by dfoxg** [reply]
Good to hear that, I was already planning to switch 🙃

For the dependency updates: maybe you can use mend renovate for that, the bot can merge all updates automatically, if the ci was running successfully. 
I don't know if you can do that with dependabot too.

**Comment by sever-sever** [comment]
BTW there are several PR from dependabot :)
What do you think about merging?
https://github.com/dswd/vpncloud/pulls?q=is%3Apr+is%3Aopen+dependabot
